src/instructlab/eval/mt_bench_judgment.py

```diff
@@ -172,15 +174,19 @@ def judge_model(
 
     question_file = f"{data_base_dir}/question.jsonl"
     answer_file = f"{output_base_dir}/model_answer/{model_name}.jsonl"
-    answer_dir = os.path.dirname(answer_file)
-    ref_answer_dir = f"{data_base_dir}/reference_answer"
+    if use_builtin_ref_answers:
+        ref_answer_file = os.path.join(data_base_dir, "reference_answer", "gpt-4.jsonl")
```

Comment:
I assume GPT-4 is the default format here due to lm_eval?

Reply:
gpt4.jsonl are the current committed reference answers for mt_bench: https://github.com/instructlab/eval/blob/main/src/instructlab/eval/data/mt_bench/reference_answer/gpt-4.jsonl

Reply:
Ack, thanks for clarifying that for me!